JavaScript/Programmes/Symbol.js

```diff
@@ -1,16 +1,17 @@
-const _password = Symbol("password");
 class User {
-  constructor(unsername, password) {
-    this.username = this.username;
-    this[_password] = password;
-  }
+  constructor(username, password) {
+  this.username = username;
+  let _password = Symbol(password)
```

Comment:
This is suppose to be `this[_password]` as _password is a Symbol and we want to use the symbol here.

Reply:
As far as I am aware, if we use this[_password] or this._password, it becomes accessible even from the outside of the User object. 
As our goal is to make the password only accessible via the getter function, I used **let _password**.

Reply:
Symbol is used to create a unique variable name i.e. key for our object...in your case, you've created a unique value to the key, which is not the intended usage.
And a key created using a symbol is not accessible using . or [] notation.